lib/lambda_ethereum_consensus/application.ex

```diff
@@ -14,18 +12,23 @@ defmodule LambdaEthereumConsensus.Application do
     {args, _remaining_args, _errors} =
       OptionParser.parse(System.argv(), switches: [checkpoint_sync: :string])
 
+    if not Keyword.has_key?(args, :checkpoint_sync) do
+      Logger.error("No checkpoint sync url provided.")
+      :init.stop(0)
```

Comment:
nit. This should be non-zero
```suggestion
      :init.stop(1)
```

Reply:
gonna add this to the follow up.